bin/dkr-build-test-container

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$( dirname "$0" )
+source "$SCRIPT_DIR/dkr-config"
+
+echo docker image rm "${CONTAINER_NAME}:${CONTAINER_VERSION}" || true 
+docker build -f docker/Dockerfile  \
+       -t "${CONTAINER_NAME}:${CONTAINER_VERSION}" .
```

Comment:
What are the content of the image supposed to be?
I'm finding hard to work out what the image should contain and what mouted?

Reply:
Principal is that container defines a software environment - most else is mounted.